app/views/sessions/new.html.erb

```diff
@@ -8,7 +8,7 @@
       <% if ENV["SIGN_IN_METHOD"] == "persona" %>
         <%= govuk_button_to t(".sign_in_with_persona"), personas_path, method: :get %>
       <% else %>
-        <%= govuk_button_to t(".sign_in_with_dsi"), "#" %>
+        <%= govuk_button_to t(".sign_in_with_dsi"), "/auth/dfe" %>
```

Comment:
We should use a path helper here.

Reply:
This path is not in our routes path, it's a path opened by the omniauth config, going to the DFE sign in form

Reply:
I see it. 🤔 Just for consistency, could we add to `routes_helper.rb`?

Maybe?

```ruby
def omniauth_sign_in_path(provider)
  "/auth/#{provider}"
end
```

Reply:
https://github.com/DFE-Digital/itt-mentor-services/pull/184/commits/812d3b46bba0af04c83bfeca3aed1d04f66bfa75